lib/matplotlib/pyplot.py

```diff
@@ -405,6 +403,10 @@ def draw_if_interactive() -> None:
     if (show is None
             or (manager_pyplot_show is not None
                 and manager_pyplot_show != base_pyplot_show)):
+        if not manager_pyplot_show:
+            raise ValueError(
+                "Backend `newbackend` defines neither `FigureCanvas.manager_class` nor "
+                "a toplevel `show` function")
```

Comment:
Maybe I am missing something obvious (or do not see properly), but do we use backticks in error messages?

Reply:
Removed them.